supportingami/sami-website
# Ft/footer

#### What does this PR do?

- Please include a summary of the change(s) and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This PR adds the footer section 

#### How should this be manually tested?

- git fetch
- git checkout origin/ft-footer
- yarn scripts strapi db:import
- Check out the footer section which is visible on all pages

#### What are the relevant GitHub issues?

None

#### Screenshots

<img width="1720" alt="Screenshot 2022-12-14 at 22 03 40" src="https://user-images.githubusercontent.com/24493616/207724912-d8f94357-a5dd-4c93-a10a-debfa6801748.png">
